medical-records/medical-records-api/src/main/java/ksch/medicalrecords/MedicalRecordEntryService.java

```diff
@@ -0,0 +1,13 @@
+package ksch.medicalrecords;
+
+import ksch.patientmanagement.visit.Visit;
+
+/**
+ * Basis for all services which deal with medical record entries.
+ *
+ * It is required to make sure that all medical record entries are linked to a {@link Visit}.
+ */
+interface MedicalRecordEntryService {
+
+    MedicalRecordEntry createMedicalRecordEntry(Visit visit);
```

Comment:
Now just `UUID`s are being used to avoid confusion. People otherwise might ask why to hand over the full object, but use only the ID for keeping the reference.

Only interacting between entities from different modules is easier to explain.

Reply:
Commit reverted as the code is nicer to read in the old way.